pom.xml

```diff
@@ -1069,6 +1070,12 @@
         <version>${zk-curator.version}</version>
       </dependency>
 
+      <dependency>
+        <groupId>com.lmax</groupId>
+        <artifactId>disruptor</artifactId>
+        <version>${disruptor.version}</version>
+      </dependency>
```

Comment:
only hudi-common needs this right ? why add it to root pom ? can you help me understand.

Reply:
no need in root. Just move it into `hudi-common`